air/src/execution_step/execution_context/last_error/last_error_definition.rs

```diff
@@ -112,3 +115,15 @@ fn ensure_jvalue_is_string(
         }),
     }
 }
+
+pub fn no_error_last_error_object() -> JValue {
+    json!({ERROR_CODE_FIELD_NAME: NO_ERROR_ERROR_CODE, MESSAGE_FIELD_NAME: NO_ERROR_MESSAGE})
```

Comment:
```suggestion
    json!({
        ERROR_CODE_FIELD_NAME: NO_ERROR_ERROR_CODE,
        MESSAGE_FIELD_NAME: NO_ERROR_MESSAGE,
    })
```

Reply:
Clippy or fmt doesn't suggest this formatting though.

Reply:
This is a code withing a non-standard macro, so fmt doesn't know how to handle it.